integration/README.md

```diff
@@ -3,45 +3,70 @@
 
 Integration tests for Cook scheduler
 
-To run the tests on a local cook install, run
+## Running in docker
+
+If you're running cook in docker, it can be beneficial to run the integration tests in a docker container so that the IP addresses resolve correctly. To do so, there are two separate scripts:
```

Comment:
Can we be more specific here? Does "it can be beneficial" mean that it's required in order for the tests to pass?

Reply:
As far as I know, this is a Mac-specific issue. Probably worth just saying that in here.

Reply:
I didn't write this (I just reordered a couple of sections in the README), but no, it's not required. I usually run Cook Scheduler in a Docker container, but run all the integration tests on my "host" machine. I think this is really talking about running Docker running on Mac/Windows, where the Docker network is isolated from the host machine.